test/views/CashAccountNoTransactionsSpec.scala

```diff
@@ -33,28 +33,48 @@ class CashAccountNoTransactionsSpec extends SpecBase {
 
   "view" should {
 
-    "display correct guidance" in new Setup {
-      val eori = "test_eori"
-      val balances: CDSCashBalance = CDSCashBalance(None)
-      val accNumber = "12345678"
-
-      val cashAccount: CashAccount = CashAccount(number = accNumber,
-        owner = eori,
-        status = AccountStatusOpen,
-        balances = balances)
-
-      val model: CashAccountViewModel = CashAccountViewModel(eori, cashAccount)
-
+    "display correct title" in new Setup {
       val viewDoc: Document = view(model)
-
       shouldContainCorrectTitle(viewDoc)
+    }
+
+    "display correct back link" in new Setup {
+      val viewDoc: Document = view(model)
```

Comment:
Could you please remove this from here and from other test cases and move to the trait Setup (and use from there) as we are using the same view (created using the same values) in all the cases?